[2024-02-27 08:53:10,818] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: triggerDagComplexa.tsk2 2024-02-27T08:52:22.410638+00:00 [queued]>
[2024-02-27 08:53:10,856] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: triggerDagComplexa.tsk2 2024-02-27T08:52:22.410638+00:00 [queued]>
[2024-02-27 08:53:10,857] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2024-02-27 08:53:10,857] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2024-02-27 08:53:10,857] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2024-02-27 08:53:10,890] {taskinstance.py:1115} INFO - Executing <Task(BashOperator): tsk2> on 2024-02-27T08:52:22.410638+00:00
[2024-02-27 08:53:10,897] {standard_task_runner.py:52} INFO - Started process 112 to run task
[2024-02-27 08:53:10,914] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'triggerDagComplexa', 'tsk2', '2024-02-27T08:52:22.410638+00:00', '--job-id', '44', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/triggerDagComplexa.py', '--cfg-path', '/tmp/tmpucmvij3b', '--error-file', '/tmp/tmp51i7pcwn']
[2024-02-27 08:53:10,915] {standard_task_runner.py:77} INFO - Job 44: Subtask tsk2
[2024-02-27 08:53:10,979] {logging_mixin.py:109} INFO - Running <TaskInstance: triggerDagComplexa.tsk2 2024-02-27T08:52:22.410638+00:00 [running]> on host 1034e60698e1
[2024-02-27 08:53:11,068] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=triggerDagComplexa
AIRFLOW_CTX_TASK_ID=tsk2
AIRFLOW_CTX_EXECUTION_DATE=2024-02-27T08:52:22.410638+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-02-27T08:52:22.410638+00:00
[2024-02-27 08:53:11,070] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2024-02-27 08:53:11,071] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 5']
[2024-02-27 08:53:11,089] {subprocess.py:74} INFO - Output:
[2024-02-27 08:53:16,093] {subprocess.py:82} INFO - Command exited with return code 0
[2024-02-27 08:53:16,146] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=triggerDagComplexa, task_id=tsk2, execution_date=20240227T085222, start_date=20240227T085310, end_date=20240227T085316
[2024-02-27 08:53:16,244] {local_task_job.py:151} INFO - Task exited with return code 0
[2024-02-27 08:53:16,300] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
